client/components/Avatar/Avatar.tsx

```diff
@@ -39,7 +39,7 @@ const Avatar = (props: Props) => {
 		borderColor,
 		borderWidth: borderColor ? borderWidth || Math.floor(width / 50) + 1 : 0,
 		fontSize: isBlock ? Math.floor(width / 1.5) : Math.floor(width / 2.5),
-		backgroundColor: communityData.accentColorDark,
+		backgroundColor: communityData.accentColorDark ?? undefined,
```

Comment:
`accentColorDark` is one that comes up particularly often, maybe should set be set to `string` in the types